css-pseudo-4/Overview.bs

```diff
@@ -593,7 +601,7 @@ Styling Highlights</h3>
 -->
 
   Note: Historically (and at the time of writing)
-  only 'color' and 'background-color' have been interoperably supported.
+  only the highlight colors have been interoperably supported.
```

Comment:
```suggestion
  only the [=highlight colors=] have been interoperably supported.
```

Reply:
done in 24df479ce2bf25d3090559715a54e7eec31d4025